docs/services/data/_index.md

```diff
@@ -91,6 +91,13 @@ Label data for management and machine learning, with dynamic datasets that chang
 
 Export data with the Viam CLI and download your data for offline access.
 
+{{% /manualcard %}}
+{{% manualcard link="/how-tos/upload-data/" %}}
+
+### Upload a batch of data
+
+Upload data to the Viam app from your local computer or mobile device using the data client API, Viam CLI, or Viam mobile app.
+
 {{% /manualcard %}}
```

Comment:
I think this should actually result in the exact same card
```suggestion
{{% card link="/how-tos/upload-data/" %}}
```

Reply:
It's actually not the same because non-manual includes the icon, and the typeface is different from the manual cards. I want to keep this uniform with the others.